### PR TITLE
Fix typos and resolve open todos

### PR DESCRIPTION
**Pull request by iamleonie**
### Type of change:

- [x] **Documentation** updates (non-breaking change to fix/update documentation)
- [ ] **Website** updates (non-breaking change to update main page, company pages, pricing, etc)
- [ ] **Content** updates – **blog**, **podcast** (non-breaking change to add/update content)
- [ ] **Bug fix** (non-breaking change to fixes an issue with the site)
- [ ] **Feature** or **enhancements** (non-breaking change to add functionality)

### How Has This Been Tested?

- [x] **Github action** – automated build completed without errors
- [x] **Local build** - the site works as expected when running `yarn start`

> note, you can run `yarn verify-links` to test site links locally

### CODE REVIEW

File: developers/academy/zero_to_mvp/103_schema_and_imports/30_import.mdx [hunk 1]
```diff
@@ -41,10 +41,6 @@ Optionally, you can manually specify:
 We'll cover these in more detail later on.
 :::
 
-:::warning TODO
```
Comment (iamleonie) [inline developers/academy/zero_to_mvp/103_schema_and_imports/30_import.mdx:44]: This is just a suggestion: I don't think an example code snippet is necessary here. The tutorial flows well without it and thus I would clean this up by simply removing this TODO field.

File: developers/academy/zero_to_mvp/103_schema_and_imports/40_example.mdx [hunk 2]
```diff
@@ -140,7 +140,7 @@ We can now add the class to the schema.
 <FilteredTextBlock
   text={PythonCodeExample}
   startMarker="# Define the class"
-  endMarker='# End class definition'
+  endMarker='# Finished creating the class'
```
Comment (databyjp) [inline developers/academy/zero_to_mvp/103_schema_and_imports/40_example.mdx:143]: Hey, this is to pull up a code snippet based on these markers. So we can change these, but we'll have to change the corresponding comment in the code. :) 

I propose to leave this one as is for now.

File: developers/academy/zero_to_mvp/104_queries_2/20_hybrid.mdx [hunk 3]
```diff
@@ -16,13 +16,13 @@ import imageUrl from '../../tmp_images/academy_placeholder.jpg';
 
 A hybrid search combines `bm25` searches that you just learned about with a vector search, producing rankings from a combination of the two results.
 
-This can produce helpful results when a vector search or a keyword search alone is not producing desired results. For example, it may be useful when a vector search alone is producing too many irrelevant results, and you want particular keywords to weight the results a certain way.
+This can produce helpful results when a vector search or a keyword search alone is not producing desired results. For example, it may be useful when a vector search alone is producing too many irrelevant results, and you want particular keywords to weigh the results a certain way.
```
Comment (databyjp) [inline developers/academy/zero_to_mvp/104_queries_2/20_hybrid.mdx:19]: I think "weight" is correct here as it's describing like scaling one of vector or bm25 results more heavily than the other.

**Review by databyjp — CHANGES_REQUESTED** [review]
Hi Leonie, thanks so much for these! :) 

Would you mind making those small changes? And then we can merge the PR.

File: developers/academy/zero_to_mvp/103_schema_and_imports/40_example.mdx (hunk 2 above)
Comment (databyjp) [reply developers/academy/zero_to_mvp/103_schema_and_imports/40_example.mdx:143]: Oh my bad ;) I see that also comes from the code :)

**Pull request merged**